samples/bookinfo/src/productpage/productpage.py

```diff
@@ -44,26 +45,29 @@
 from flask_bootstrap import Bootstrap
 Bootstrap(app)
 
+servicesDomain = "" if (os.environ.get("SERVICES_DOMAIN") == None) else "." + os.environ.get("SERVICES_DOMAIN")
+servicesPort = "9080"
+
 details = {
-    "name" : "http://details:9080",
+    "name" : "http://{0}{1}:{2}".format("details", servicesDomain, servicesPort),
```

Comment:
nit, more clear like this: "http://details{0}:{1}".format(servicesDomain, servicesPort)